### PR TITLE
Issue127

### PR DESCRIPTION
**Pull request by shigdel**
renamed and changed the content   CalculatedUnifrac to CalculatedUnifrac

### CODE REVIEW

**Review by ChouaibB — CHANGES_REQUESTED** [review]
File should be renamed as "calculateUnifrac"

**Review by ChouaibB — APPROVED** [review]

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
Hi, thanks for the PR!

Now you have two pull requests open for this same thing. In addition to this, you have opened PRs https://github.com/microbiome/mia/pull/153 and https://github.com/microbiome/mia/pull/129 (already closed) which are related to the issue https://github.com/microbiome/mia/issues/127

It is better to have one PR for one thing to simplify things. Because now some discussions are in  another PR, and this is some kind of documentation about what we have done and why we haven't done some things. 

If you want to revert / undo things that you have made to your branch, that is always possible

**So, for the future PRs: one issue = one PR is better**

Also, you are merging this to `microbiome:issue127` branch and not to `master`. Is there any reason for that? **I think you should merge this directly into the master.** 

That can be easily changed: 

1. press edit (next to the PR title)
2. change base: issue127 to master

**You haven't generated manual page based on your changes. So you can do that by running `devtools::document()`**, to see if there is any errors.

Otherwise looks good

-Tuomas

**Comment by antagomir** [comment]
I would support that only specific core maintainers will merge PRs to master.

**Comment by TuomasBorman** [comment]
> I would support that only specific core maintainers will merge PRs to master.

Well that might be actually better for example because of the limitations of testing when project is forked

**Review by TuomasBorman — APPROVED** [review]

**Comment by FelixErnst** [comment]
Please add a new generic for the old name, add a deprected note, specifically explaining the tiny renaming part. Thanks

**Comment by antagomir** [comment]
@shigdel let us know if there are any problems with implementing the proposed changes -

**Comment by FelixErnst** [comment]
This PR is not aimed at the master branch. Are you aware of that?

**Comment by TuomasBorman** [comment]
> This PR is not aimed at the master branch. Are you aware of that?

Yes, that is intentional. This allows, e.g., us to test it before merging into master

**Comment by TuomasBorman** [comment]
Maybe best to revert that sync, because this is quite a mess

File: R/calculateUnifrac.R
Comment (TuomasBorman) [inline R/calculateUnifrac.R:158]: message

**Comment by TuomasBorman** [comment]
Branch `issue127_new` includes these changes without conflicts. I'm not sure what would have been the best practice in this case, because now it shows only myself as a contributor

**Comment by FelixErnst** [comment]
Can you update from master, to get a current idea of the changes? Thanks

**Comment by TuomasBorman** [comment]
How this should be done correctly? 

If I update `shigdel/issue127` from `microbiome/master` over 50 commits appears, because it is not synced. At least last time it showed that all those all 50 commits were new modifications. 

Or should we continue with `issue127_new` branch? This kind of git problems are not too familiar to me

**Comment by antagomir** [comment]
If this is just about renaming UniFrac -> Unifrac, perhaps easier to redo all with a clean branch?

**Pull request closed without merging**
